app/vminsert/promremotewrite/request_handler.go

```diff
@@ -70,6 +75,57 @@ func insertRows(timeseries []prompb.TimeSeries, extraLabels []prompbmarshal.Labe
 				return err
 			}
 		}
+		exemplars := ts.Exemplars
+		for i := range exemplars {
```

Comment:
Lets remove exemplars push for now. It's incorrect to treat is as a metric.

Reply:
removed